test/IFrame.spec.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  browser
+} from 'protractor';
+
+import {
+  IFramePage
+} from '../src/page/Iframe.page';
+
+describe('open browser', () =>{
+
+  beforeEach(async () => {
+    await browser.get(
+      'http://toolsqa.com/iframe-practice-page/'
+    );
+  });
+
+  describe('checking global title', () => {
+    const checkingGlobalName: IFramePage = new IFramePage ();
+
+    it('checking global title', async () => {
+      expect(checkingGlobalName.getPageName())
+      .toBe('IFrame practice page');
+    });
+
+    describe('checking iframe title', () =>{
+      beforeAll(async () => {
+        await checkingGlobalName.goToIFrameOne();
```

Comment:
Aquí puedes guardar en una variable el valor del título para compararlo en el it